addons/dynamishelper/dynamishelper.lua

```diff
@@ -24,408 +24,427 @@
- 			if mob == staggers[window][proctype[i]][j] then
- 				current_proc = proctype[i]
- 			end
- 		end
- 	end
- 	if current_proc == 'ja' then
- 		current_proc = 'Job Ability'
- 	elseif current_proc == 'magic' then
- 		current_proc = 'Magic'
- 	elseif current_proc == 'ws' then
- 		current_proc = 'Weapon Skill'
- 	end
-	initializeproc()
+    local window
+    if currenttime >= 0*60 and currenttime < 8*60 then
+        window = 'morning'
+    elseif currenttime >= 8*60 and currenttime < 16*60 then
+        window = 'day'
+    elseif currenttime >= 16*60 and currenttime <= 24*60 then
+        window = 'night'
+    end
+
+    for i=1, #proctype do
+        for j=1, #staggers[window][proctype[i]] do
+            if current_mob == staggers[window][proctype[i]][j] then
+                current_proc = proctype[i]
+            end
+        end
+    end
+    if current_proc == 'ja' then
+        w.current_proc = 'Job Ability'
+    elseif current_proc == 'magic' then
+        w.current_proc = 'Magic'
+    elseif current_proc == 'ws' then
+        w.current_proc = 'Weapon Skill'
+    end
 end
 
-function initializeproc()
---		print('initializeproc function')
-		windower.text.create('proc_box')
-	 	windower.text.set_bg_color('proc_box',200,30,30,30)
-	 	windower.text.set_color('proc_box',255,200,200,200)
-	 	windower.text.set_location('proc_box',pposx,pposy)
-	 	if proc == 'on' then
-	 	 	windower.text.set_visibility('proc_box', true)
-	 	end
-	 	windower.text.set_bg_visibility('proc_box',1)
-	 	windower.text.set_font('proc_box','Arial',12)
-	 	windower.text.set_text('proc_box',' Current proc for \n '..mob..'\n is '..current_proc);
-	 	if proc == "off" then
-	 		windower.text.set_visibility('proc_box', false)
-	 	end
+-------------------------------------------------------------------------------
+-- Print the help Information -------------------------------------------------
+-------------------------------------------------------------------------------
+help = T{
+size = "Usage: dh size [font size] - Your current size is %u pixel",
+font = "Usage: dh font [font name] - You are currently using %q",
+opacity = "Usage: dh opacity [0-100]%% - Opacity is currently at %u%%",
+padding = "Usage: dh padding [size] - Padding is currently %u pixels",
+bgcolor = "Usage: dh bgcolor [Red] [Green] [Blue] - Background color - Example: 255 0 0 is red",
+stroke = "Usage: dh stroke [width] - Stroke width is currently %u pixels",
+stopacity = "Usage: dh stopacity [0-100]%% - Stroke opacity is currently at %u%%",
+stcolor = "Usage: dh stcolor [Red] [Green] [Blue] - Stroke color - Example: 255 0 0 is red",
+posx = "Usage: dh posx [x] - Current window position is x=%u",
+posy = "Usage: dh posy [y] - Current window position is y=%u",
+ll = "Usage: dh ll create: Creates a light luggage profile to lot all dynamis currency.'"
+}
+
+function printHelp(command,val)
+    if not command and not val then
+        windower.add_to_chat(159,'\nDynamisHelper v2.0')
```

Comment:
`2.0` is not quite right. You can just use `'DynamisHelper v' .. tostring(_addon.version)`.